src/style.css

```
@@ -0,0 +1,133 @@
+.page {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 300;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  width: 198px;
+  height: 406px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 40px;
```

What if the card width will change? You will need to update these margins too
And check 10 point from [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)

```suggestion
  margin: 32px auto 40px;
```